bash2048.sh

```diff
@@ -262,6 +264,70 @@ function reload_game {
   index_max=board_size-1
 }
 
+# print the content of the highscores file
+function show_highscores {
+    output="\nHighscore Name Date\n\n"
+    while read line; do
+        output="${output}${line}\n"
+    done < "${highscore_file}"
+    printf "${output}" | column -t
+}
+
+# check if a score is a high score (in top 10 highest scores)
+# input:
+#         $1: score value
+# output:
+#         0 if true
+#         1 if false
+function is_high_score(){
```

Comment:
not sure if this is intended, but the new highscore is saved only if it is greater than lowest previous highscore, so if I have one highscore of 17000, my second game with 16900 would not qualify as highscore even if there is still space in top $highscore_count_max games

Also another check for highscore file existence should be here.

Reply:
Yes for the file existence check, I added it.
Concerning new high scores, the else branch in 
https://github.com/mydzor/bash2048/pull/27/files#diff-c6940b32399efce06568ae3f7ec981e4R310-R316
makes sure a high score is added when there is still room for it regardless of the score.